src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -24,9 +26,39 @@
  *
  * @author Kuanhung Chen <ericj.tw@gmail.com>
  */
-final class MethodArgumentSpaceFixer extends AbstractFixer
+final class MethodArgumentSpaceFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
     /**
+     * Preserve existing multiple spaces after comma.
+     *
+     * @var bool
+     */
+    private $multipleSpacesAfterComma = false;
+
+    /**
+     * @param null|array $configuration
+     *
+     * @throws InvalidFixerConfigurationException
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            return;
```

Comment:
please read the docs of the interface you are implementing.
`null` has to reset the configuration, not `do nothing`